index.html

```diff
@@ -5,23 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="css/all.css">
     <title>Document</title>
 </head>
 <body>
-    <header>
-        <ul class="nav-bar">
-            <li><a href="#">Porfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">info</a></li>
-        </ul>
-    </header>
+    <section>
+        <nav class="nav">
+            <ul class="nav-bar">
+                <li><a href="#">Porfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">info</a></li>
+            </ul>
+         </nav>
+        </section>
     <main class="main-content">
-        <h1 class="heading">Hey there I,m Bunbee.<br>I'm a software developer</h1>
+        <h1 class="heading">Hey there I,m Bunbee.<br> <span class="green">I'm a software developer</span></h1>
         <p>I can help you build a product, feature or website. Look thrrough some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
         <ul class="footer">
-            <li><a href="https://github.com/mutinhiri">github</a></li>
-            <li><a href="www.linkedin/mutinhiri">Linkedin</a></li>
-            <li><a href="twitter.com/bunbee">Twitter</a></li>
+            <li><a href="https://github.com/mutinhiri"><i class="fab fa-instagram"></i></a></li>
+            <li><a href="www.linkedin/mutinhiri"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="twitter.com/bunbee"><i class="fab fa-github"></i></a></li>
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
```

Comment:
thank you for noting that. however i have added the icons using font awesome i guess it will e different with the remote version. My local version is showing all the icons. can you please advise on a better way to implement the icons